devito/arch/compiler.py

```diff
@@ -400,6 +400,17 @@ def __lookup_cmds__(self):
         self.MPICXX = 'mpicxx'
 
 
+class ArmCompiler(GNUCompiler):
```

Comment:
How is this different from previous ARM support?

Reply:
We did not have an ARM compiler before only gcc, this adds the extra flags for graviton and I'm sure other "brands" will need some flags too.